<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Heidenhersh</class>
 <widget class="QWidget" name="Heidenhersh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="hlPreset">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbPreset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttNewPreset">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttSavePreset">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttRemovePreset">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="txtInput"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlText">
     <item>
      <layout class="QHBoxLayout" name="hlHeight">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dspnTextHeight">
         <property name="suffix">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>30.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="chkMirror">
       <property name="text">
        <string>Mirror</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlSpacing">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Spacing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dspnSpacing">
         <property name="suffix">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlCutting">
     <item>
      <layout class="QHBoxLayout" name="hlNCuts">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>N Cuts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnNCuts">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlFeed">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Feed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnFeed">
         <property name="suffix">
          <string>mm/min</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>6000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>FMAX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnFmax">
         <property name="suffix">
          <string>mm/min</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>6000</number>
         </property>
         <property name="singleStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlZParams">
     <item>
      <layout class="QHBoxLayout" name="hlZSafePos">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Z Safe pos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dspnZSafePos">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-300.000000000000000</double>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.200000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlWorkZPos">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Workpiece Z pos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dspnWorkpZPos">
         <property name="minimum">
          <double>-300.000000000000000</double>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlDepth">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Depth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dspnDepth">
         <property name="suffix">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRotary">
     <property name="text">
      <string>Rotary</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameRotary">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="hlWheelDia">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Wheel dia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dspnWheelDia">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>160.000000000000000</double>
          </property>
          <property name="value">
           <double>60.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="chkGroove">
        <property name="text">
         <string>Project on groove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameGroove">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="hlGroove">
           <item>
            <layout class="QHBoxLayout" name="hlCenterZOffs">
             <item>
              <widget class="QLabel" name="lblGrooveCenterZOffs">
               <property name="text">
                <string>Center Z offset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dspnGrooveZOffs">
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="minimum">
                <double>-300.000000000000000</double>
               </property>
               <property name="maximum">
                <double>300.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hlGrooveRadius">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Groove radius</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dspnGrooveRadius">
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>20.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>2.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hlSegLen">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Max segment length</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dspnGrooveSegLen">
               <property name="suffix">
                <string>mm</string>
               </property>
               <property name="minimum">
                <double>0.100000000000000</double>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.200000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlButtons">
     <item>
      <widget class="QPushButton" name="buttGenerate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttSimulate">
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>txtInput</zorder>
  <zorder>frameRotary</zorder>
  <zorder>chkRotary</zorder>
 </widget>
 <tabstops>
  <tabstop>txtInput</tabstop>
  <tabstop>dspnTextHeight</tabstop>
  <tabstop>chkMirror</tabstop>
  <tabstop>dspnSpacing</tabstop>
  <tabstop>spnNCuts</tabstop>
  <tabstop>spnFeed</tabstop>
  <tabstop>dspnZSafePos</tabstop>
  <tabstop>dspnWorkpZPos</tabstop>
  <tabstop>dspnDepth</tabstop>
  <tabstop>chkRotary</tabstop>
  <tabstop>dspnWheelDia</tabstop>
  <tabstop>chkGroove</tabstop>
  <tabstop>dspnGrooveZOffs</tabstop>
  <tabstop>dspnGrooveRadius</tabstop>
  <tabstop>dspnGrooveSegLen</tabstop>
  <tabstop>buttGenerate</tabstop>
  <tabstop>buttSimulate</tabstop>
  <tabstop>cmbPreset</tabstop>
  <tabstop>buttNewPreset</tabstop>
  <tabstop>buttSavePreset</tabstop>
  <tabstop>buttRemovePreset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
